src/lib/database/seed.ts

```diff
@@ -8,9 +8,11 @@ export interface Seed {
   august_device_1: "august_device_1"
   august_device_2: "august_device_2"
   schlage_device_1: "schlage_device_id"
+  seam_seedkey1_seedkey1: "seam_seedkey1_seedkey1"
+  seam_seedkey2_seedkey2: "seam_seedkey2_seedkey2"
 }
 
-export const seed = (db: Database): Seed => {
+export const seed = (db: DatabaseState & DatabaseMethods): Seed => {
```

Comment:
Odd, `Database` should be correct here. Can I ask what the type error was?

Reply:
the db passed into seed sometimes doesn't have `getState` and `setState` if it comes from the fake IIRC. This is a bit narrower so IMO it's ok, broadening types is generally worrying

Reply:
Ah, I see! Yea actually we should probably just update `Database` to be the same as here: https://github.com/seamapi/fake-seam-connect/blob/60f9a0548f397ed85c52d24a091638c8803d25c1/src/lib/fake.ts#L11

That update I made felt like a hack. If we need to use the object with the zustand methods it should be an internal type (ZustandDatabase or Store?).

Reply:
544eb50